src/server.h

```diff
@@ -1181,6 +1181,26 @@ typedef struct ClientModuleData {
                                                 * unloaded for cleanup. Opaque for the Server Core.*/
 } ClientModuleData;
 
+/* Parser state and parse result of a command from a client's input buffer. */
+typedef struct commandParserState {
```

Comment:
```suggestion
typedef struct parsedCommand {
```
Maybe this naming made sense earlier, but it seems to imply the orchestrator and not an individual parsed command.

Reply:
Sgtm. This is used also for partially parsed commands, but that's a side minor detail.